2996_reflection/reflection.md

```diff
@@ -6767,6 +6767,100 @@ consteval source_location source_location_of(info r);
 
 ::: std
 ::: addu
+```cpp
+consteval bool $has-type$(info r); // exposition only
+```
+
+[#]{.pnum} *Returns*: `true` if  `r` represents a value, object, variable, function that is not a constructor or destructor, enumerator, non-static data member, bit-field, direct base class relationship, or data member description. Otherwise, `false`.
```

Comment:
pre-existing: should be unnamed bit-field?

Reply:
Done